docs/examples/frameworks/tensorflow/tensorflow-dataset-multigpu.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>To make the training distributed to multiple GPUs,  we use <code>tf.distribute.MirroredStrategy.</code> Comma?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done